src/common/components/button/Button.tsx

```diff
@@ -1,28 +1,69 @@
-import React, { FunctionComponent, ReactNode } from 'react';
-import joinClassNames from 'classnames';
+import React from 'react';
+import { Button as HdsButton, ButtonProps } from 'hds-react';
 
-import styles from './button.module.scss';
+const kukkuuSecondaryButtonStyles = {
+  '--background-color': 'var(--color-white)',
```

Comment:
I didn't know this syntax was possible. Looks good to me, but I can't give a good opinion about it.